src/components/navbar/Navbar/ProfileModal.tsx

```diff
@@ -6,57 +6,54 @@ import AddressAvatar from '@/components/AddressAvatar'
 import Button from '@/components/Button'
 import { CopyText, CopyTextInline } from '@/components/CopyText'
 import Modal, { ModalFunctionalityProps } from '@/components/Modal'
+import { ACCOUNT_SECRET_KEY_URL_PARAMS } from '@/pages/account'
 import { useSendEvent } from '@/stores/analytics'
 import { useMyAccount } from '@/stores/my-account'
-import { truncateAddress } from '@/utils/account'
+import { decodeSecretKey, truncateAddress } from '@/utils/account'
 import { cx } from '@/utils/class-names'
 import { getBaseUrl } from '@/utils/env/client'
 import React, { useEffect, useState } from 'react'
-import { HiOutlineChevronLeft } from 'react-icons/hi'
 import QRCode from 'react-qr-code'
 import urlJoin from 'url-join'
```

Comment:
Is it really need another library?

Reply:
I have also changed most icons to be react-icons/hi2 in [cb3bb2a](https://github.com/dappforce/grillchat/pull/18/commits/cb3bb2ae21d41781e7fb7d32226f1d39baa63250) except the checkmark because there is no icon like that in hi2

Reply:
I asked about `url-join` )

Looks really easy lib, but for now doesn't make sense remove it